zebra/zebra_fpm_netlink.c

```diff
@@ -254,13 +254,62 @@ static int netlink_route_info_add_nh(struct netlink_route_info *ri,
 static uint8_t netlink_proto_from_route_type(int type)
 {
 	switch (type) {
-	case ZEBRA_ROUTE_KERNEL:
```

Comment:
I think we could reuse `zebra2proto()`.

Reply:
@ton31337: Agree, pushed a latest patch with changes.